pgzero/storage.py

```diff
@@ -0,0 +1,114 @@
+import json
+import os
+import platform
+import logging
+
+
+logger = logging.getLogger(__name__)
+
+
+__all__ = ['StorageCorruptionException', 'JSONEncodingException', 'Storage']
+
+
+class StorageCorruptionException(Exception):
+    """The data in the storage is corrupted."""
+
+
+class JSONEncodingException(Exception):
+    """The data in the storage is corrupted."""
+
+
+class Storage(dict):
+
+    path = ''
+
+    """Behaves like a dictionary with a few extra functions.
+
+    It's possible to load/save the data to disk.
+
+    The name of the file will be the script's name hashed.
+
+    NOTE: If two scripts have the same name, they will load/save from/to
+    the same file.
+    """
+    def __init__(self):
+        dict.__init__(self)
+
+        storage_path = self._get_platform_pgzero_path()
+        if not os.path.exists(storage_path):
+            os.makedirs(storage_path)
+
+    @classmethod
+    def set_app_hash(cls, name):
+        storage_path = cls._get_platform_pgzero_path()
+        cls.path = os.path.join(storage_path, '{}.json'.format(name))
+
+    def load(self):
+        """Load data from disk."""
+        self.clear()
+
+        try:
+            with open(self.path, 'r') as f:
+                data = json.load(f)
+        except FileNotFoundError:
+            # If no file exists, it's fine
+            logger.debug('No file to load data from.')
+        except json.JSONDecodeError:
+            msg = 'Storage is corrupted. Couldn\'t load the data.'
```

Comment:
You should use double-quotes for human readable strings so that you don't have to escape apostrophes :-)